test/tests/options.rb

```diff
@@ -0,0 +1,291 @@
+    assert options[:url_safe_methods] == Set.new([:test_method1, :test_method2])
+  end
+
+  def test__skip_file_option
+    options = setup_options_from_input("--skip-files", "file1.rb,file2.rb,file3.js,file2.rb")
+    assert options[:skip_files] == Set.new(["file1.rb", "file2.rb", "file3.js"])
+  end
+
+  def test_only_files_option
+    options = setup_options_from_input("--only-files", "file1.rb,file2.rb,file3.js,file2.rb")
+    assert options[:only_files] == Set.new(["file1.rb", "file2.rb", "file3.js"])
+  end
+
+  def test_add_lib_paths_option
+    options = setup_options_from_input("--add-libs-path", "../tests/,/badStuff/hackable,/etc/junk,../tests/")
+    assert options[:additional_libs_path] == Set.new(["../tests/", "/badStuff/hackable", "/etc/junk"])
+  end
+
+  def test_run_checks_option
+    options = setup_options_from_input("-t", "CheckSelectTag,CheckSelectVulnerability,CheckSend,CheckSelectTag,I18nXSS")
+    assert options[:run_checks] == Set.new(["CheckSelectTag", "CheckSelectVulnerability", "CheckSend", "CheckI18nXSS"])
+
+    options = setup_options_from_input("--test", "CheckSelectTag,CheckSelectVulnerability,CheckSend,CheckSelectTag,I18nXSS")
+    assert options[:run_checks] == Set.new(["CheckSelectTag", "CheckSelectVulnerability", "CheckSend", "CheckI18nXSS"])
+  end
+
+  def test_skip_checks_option
+    options = setup_options_from_input("-x", "CheckSelectTag,CheckSelectVulnerability,CheckSend,CheckSelectTag,I18nXSS")
+    assert options[:skip_checks] == Set.new(["CheckSelectTag", "CheckSelectVulnerability", "CheckSend", "CheckI18nXSS"])
+
+    options = setup_options_from_input("--except", "CheckSelectTag,CheckSelectVulnerability,CheckSend,CheckSelectTag,I18nXSS")
+    assert options[:skip_checks] == Set.new(["CheckSelectTag", "CheckSelectVulnerability", "CheckSend", "CheckI18nXSS"])
+  end
+
+  def test_add_checks_paths_option
+    options = setup_options_from_input("--add-checks-path", "../addl_tests/")
+    local_path = File.expand_path('../addl_tests/')
+    assert options[:additional_checks_path] == Set.new(["#{local_path}"])
+  end
+
+  def test_format_options
+    format_options = {
+      pdf: :to_pdf,
+      text: :to_s,
+      html: :to_html,
+      csv: :to_csv,
+      tabs: :to_tabs,
+      json: :to_json,
+      markdown: :to_markdown,
+      codeclimate: :to_codeclimate,
+      cc: :to_cc
+    }
+
+    format_options.each_pair do |key, value|
+      options = setup_options_from_input("-f", "#{key}")
+      assert options[:output_format] == value, "Expected #{key} to be #{value}."
+    end
+
+    format_options.each_pair do |key, value|
+      options = setup_options_from_input("--format", "#{key}")
+      assert options[:output_format] == value, "Expected #{key} to be #{value}."
+    end
+  end
+
+  def test_CSS_file_option
+    options = setup_options_from_input("--css-file", "../test.css")
+    local_path = File.expand_path('../test.css')
+    assert options[:html_style], local_path
```

Comment:
Did you mean to use `assert_equal` here (and in a bunch of other places below)? The second argument to `assert` is the failure message.